src/_pytest/_io/saferepr.py

```diff
@@ -1,8 +1,28 @@
+import pprint
 import sys
 
 from six.moves import reprlib
 
 
+def _safe_format_call(call, x, *args):
```

Comment:
This should be renamed to something more explicit like `_call_and_format_exception`.